stable/kong/templates/admission-webhook.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.ingressController.admissionWebhook.enabled }}
+{{- $cn := printf "%s-validation-webhook.%s.svc" ( include "kong.fullname" . ) .Release.Namespace }}
```

Comment:
Can we move this to a `_helper`? This is buried here and historically we have had problems with such settings which get overlooked.

Reply:
I wasn't sure exactly which part you wanted moved to a helper, so i moved just the service name and referenced it here, the Service resource's name, and the webhook definition's service name